repair/row_level.cc

```diff
@@ -1878,7 +1878,7 @@ class repair_meta {
                 if (status_opt) {
                     repair_stream_cmd status = std::move(std::get<0>(status_opt.value()));
                     rlogger.trace("put_row_diff: Got status code from follower={} for put_row_diff, status={}", remote_node, int(status));
-                    if (status == repair_stream_cmd::put_rows_done) {
+                    if (status == repair_stream_cmd::put_rows_done || status == repair_stream_cmd::table_dropped) {
```

Comment:
This will technically do the job, but I think at this point we should be bailing out of this repair, because it is clear that the remote doesn't have this table anymore.

Reply:
Can there be a situation where this node sees a table but the other one doesn't see it **yet**?

Reply:
I think so. But in that case the repair will be incomplete, we can drop an arbitrary amount of the data before the node notices that the table was added.

@asias what do you think?

Reply:
But if we stop the repair for that table, because one receiver does not know about it, the other nodes (which know about the table) won't get it either.

Reply:
I see, good point.

Reply:
> Can there be a situation where this node sees a table but the other one doesn't see it yet?

Yes.  This is the whole point we need to send back the no such table status code to the master node in the first place.